### ISSUE TITLE
Enhancements to Installation Steps

### ISSUE THREAD

**Issue by MariaTaborda** [nextcloud/documentation#1478]
I was trying to read through the user install documentation and found that the windows section could use some help (I found it hard to follow). One initial thought is that it shouldn't be embedded under the Installing with Linux section, the other edits are int he picture below. 

The way this is indented and the lack of numbering makes it hard to know when a part of the section ends and another one begins.
 
![Suggested Edits](https://user-images.githubusercontent.com/16101983/59142474-b1dff680-8973-11e9-9a80-f5ae46dd894a.JPG)

**Comment by skjnldsv** [comment]
Hello @MariaTaborda 
Thanks a lot for your feedback!

Could you maybe help us a bit?
You can find the edit button on top of the pages :)
https://docs.nextcloud.com/server/stable/admin_manual/installation/source_installation.html#vm-label

**Comment by MariaTaborda** [comment]
Hi @skjnldsv I would love to take a stab at it, although I am still a bit confused, about the steps myself, let me play around with it and I will come here or the forum if I have follow up questions. Thanks!

**Comment by skjnldsv** [comment]
Awesome!! Sure, I'll be happy to assist you! :v:

**Comment by skjnldsv** [comment]
The Windows installation section has been restructured since this was filed. It is now its own clearly labeled section at https://docs.nextcloud.com/server/latest/admin_manual/installation/source_installation.html#installing-on-windows-virtual-machine
